lta/lta_cmd.py

```diff
@@ -191,17 +288,32 @@ def date_ok(d: str) -> bool:
                                        action="store_true")
     parser_request_status.set_defaults(func=request_status)
 
+    # define a subparser for the 'start' subcommand
+    parser_start = subparser.add_parser('start', help='start the component')
+    parser_start.add_argument("component",
+                              choices=COMPONENT_NAMES,
+                              help="LTA component")
+    parser_start.set_defaults(func=start)
+
     # define a subparser for the 'status' subcommand
     parser_status = subparser.add_parser('status', help='perform a status query')
     parser_status.add_argument("component",
-                               choices=["picker", "bundler"],
+                               choices=COMPONENT_NAMES,
                                help="optional LTA component",
                                nargs='?')
     parser_status.add_argument("--json",
                                help="display output in JSON",
                                action="store_true")
     parser_status.set_defaults(func=status)
 
+    # define a subparser for the 'stop' subcommand
+    parser_stop = subparser.add_parser('stop', help='stop the component')
+    parser_stop.add_argument("component",
+                             choices=COMPONENT_NAMES,
+                             help="optional LTA component",
+                             nargs='?')
```

Comment:
If we're just going to ignore the component name, why take it as an argument?

Reply:
I ended up making this component argument non-optional.
On the upside, multiple components can be started from the same repo directory.